java/code/src/com/redhat/rhn/manager/system/VirtualizationActionCommand.java

```diff
@@ -95,7 +99,15 @@ public ValidatorError store()
         LOG.debug("creating virtAction");
         BaseVirtualizationAction virtAction =
             (BaseVirtualizationAction) ActionFactory.createAction(this.getActionType());
-        virtAction.setName(this.getActionType().getName());
+        String actionName = this.getActionType().getName().replaceAll("\\.$", "");
```

Comment:
I assume this just removes the '.' at the end. Can you give an example of why this is needed?

Reply:
Because that string is stored in the DB (see `schema/spacewalk/common/data/rhnActionType.sql`) and finishes with a `.` and the code further down adds the name of the VM to this string.

I know this is super bad localization practice... but to get better we would need to remove all localized strings that are in the DB ;)

Reply:
Fair enough :)